src/sagas/task.saga.js

```diff
@@ -134,6 +134,18 @@ export function* launchDevServer({ task }: Action): Saga<void> {
   }
 }
 
+export function runInstall(installCommand: any): Promise<void> {
+  return new Promise((resolve, reject) => {
+    installCommand.on('exit', (code: number, signal: string) => {
+      if (code === 0) {
+        resolve(undefined);
+      } else {
+        reject(new Error('Exit with error code: ' + code));
+      }
+    });
+  });
+}
```

Comment:
Clever solution! Shows how convenient generators are, where we can create code that behaves as though it's synchronous, without blocking the main thread.

Can we rename it `waitForInstallationToComplete`? `runInstall` makes it sound like it's actually running the command, instead of just listening for an exit code.

Reply:
This also seems like kind of a convenient helper... I wonder if it's worth generalizing it, naming it something like 'waitUntilChildProcessToComplete`?

Probably premature at this point, but I'm gonna keep an eye out, if we see we need this behaviour elsewhere we can make it a generic helper

Reply:
`waitUntilChildProcessToComplete` is good, much more descriptive. Naming things has never been my strength... 😆

Reply:
Should I break it out into utils (or another file)?